astropy/io/ascii/qdp.py

```diff
@@ -138,7 +138,7 @@ def _get_lines_from_file(qdp_file):
         lines = qdp_file.split("\n")
     elif isinstance(qdp_file, str):
         with open(qdp_file) as fobj:
-            lines = [line.strip() for line in fobj.readlines()]
+            lines = [line.strip() for line in fobj]
```

Comment:
In this case it passes tests with `line.rstrip("\n")`, so (leading) blanks are not an issue.

Reply:
Without getting into a detailed review of the code and format specs, we cannot assumes that the test inputs fully cover all possibilities. In other words, in a PR like this we really should stick with changes that do not obviously impact the code behavior.

Reply:
This refactor was moved to #16656. Please consider closing this thread.